hokusai/production.yml

```diff
@@ -87,6 +87,7 @@ metadata:
     service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
     service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"
     service.beta.kubernetes.io/aws-load-balancer-connection-draining-enabled: "true"
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "5"
```

Comment:
To true up to template: https://github.com/artsy/artsy-hokusai-templates/blob/2070773efe4bfd89ba3f4ef82eb02ac20f3a21f6/nodejs/hokusai/staging.yml.j2#L146